### ISSUE TITLE
Modules are initialized twice?

### ISSUE THREAD

**Issue by leonard-gleyzer** [patrick-kidger/equinox#368]
Hello,

I'm not sure if this is a bug or something I'm not understanding properly, but it seems that modules initialize twice?

MWE:

```python
import equinox as eqx

class Mod(eqx.Module):
    
    def __init__(self):
        print('Module Initialized')
        
Mod()
```

prints

```
Module Initialized
Module Initialized 
```

I have a Module with various time-consuming components in initialization, and everything is running twice, doubling the execution time.

**Comment by patrick-kidger** [comment]
Thanks for the report! Should be fixed in #369.

**Comment by patrick-kidger** [comment]
Closing as it's now fixed!
